components/nifosg/nifloader.cpp

```diff
@@ -84,82 +84,6 @@ namespace
             collectDrawableProperties(nifNode->parent, out);
     }
 
-    class FrameSwitch : public osg::Group
```

Comment:
FrameSwitch and BillboardCallback were just moved into the header so that I could reference them from serializers via #include.  It's possible that these nodes don't need to be exported at all, but I haven't figured out how to mark entire nodes as not needing to be exported, so they generate warnings if we don't implement an empty serializer for them.